docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/introduction.md

```diff
@@ -0,0 +1,38 @@
+**September 2023 - Release 1.0**
+
+**The Git-based process you need for mainframe development**
+
+**(Draft Document Sept 1,2023)**
+
+
+
+**Table of contents**
+
+[1 Introduction](#introduction)
+
+[2 The Git-based development process for Mainframe development](the-git-based-development-process-for-mainframe-development.md#the-git-based-development-process-for-mainframe-development)
+
+[3 Pipeline design and implementation supporting the workflows](pipeline-design-and-implementation-supporting-the-workflows.md#pipeline-design-and-implementation-supporting-the-workflows)
+
+[Conclusion](conclusion.md#conclusion)
+
+[4 Appendix](appendix.md#appendix)
+
+# Introduction
+
+Git branching strategies are patterns to enable development teams using Git to manage their code in a streamlined manner. Since Git is established as a de-facto standard for source-code management in the developers community, several approaches were designed to fulfill users' requirements and manage applications source code lifecycle, with advantages and drawbacks depending on use cases. Based on the experience gained designing branching strategies, this document describes a blueprint implementation of mainline-based development approach for mainframe applications using feature branches with an early integration pattern. This setup leverages a standardized development toolset based on an enterprise-wide Git provider and a Continuous Integration/Continuous Deployment (CI/CD) toolchain.
+
+A characteristic of this integration pattern is that developers are implementing changes for a planned release and integrate their changes into a common permanent branch (the shared configuration) that is built, tested, and released together as one consistent entity.
+
+The purpose of streamlining both the DevOps solutions and the delivery workflow is to simplify the process for development teams to deliver quality product releases on time. This enables agile development practices that allow the teams to respond more effectively to changes in the market and customer needs. The first part of the document introduces the model and outlines the development workflow from the developer's perspective. The details of the technical implementation with IBM Dependency Based Build and zAppBuild as well as packaging and deployment are discussed in the second part. All branching models are adaptable to the needs of specific teams and their applications. This document advocates for best practices and indicates where variations can be applied.
+
+Some aims and assumptions which guided our recommendations include:
+
```

Comment:
adding emphasis that this process both scales up to large and busy teams as well as scaling down to smaller teams with less frequent changes in #417